### ISSUE TITLE
Add IFS/HTESSEL grib codes for vegetation related variables

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#802]
The request from Annalisa, see [ECE portal issue 1312-106](https://dev.ec-earth.org/issues/1312#note-106):

The output coming from the grib codes for the following IFS/HTESSEL variables is requested: 27(cvl), 28(cvh), 29(tvl), 30(tvh), 66(lai_lv), 67(lai_hv) at `6-hrs frequency` is fine (as we will need them monthly).

Instead of adding a special list of grib codes with some adhoc special infrastructure for that, it is easier for the `genecec` system just to add these variables (although there is no need from the side of the requesting people), with that it is also possible to cmorise (including remapping) these variables allowing a quick and easy glance on them.

 | grib code                                          | longname                                    | CMOR table - variable  | based on           |
 | -------------------------------------------------- | ------------------------------------------- | ---------------------- | ------------------ |
 | [27.128](https://codes.ecmwf.int/grib/param-db/27) | Low  vegetation cover (`cvl`)               | `HTESSSELmon` `cvl`    | `Lmon` `c3PftFrac` |
 | [28.128](https://codes.ecmwf.int/grib/param-db/28) | High vegetation cover (`cvh`)               | `HTESSSELmon` `cvh`    | `Lmon` `c3PftFrac` |
 | [29.128](https://codes.ecmwf.int/grib/param-db/29) | Type of low vegetation (`tvl`)              | `HTESSSELmon` `tvl`    | `Lmon` `c3PftFrac` |
 | [30.128](https://codes.ecmwf.int/grib/param-db/30) | Type of high vegetation (`tvh`)             | `HTESSSELmon` `tvh`    | `Lmon` `c3PftFrac` |
 | [66.128](https://codes.ecmwf.int/grib/param-db/66) | Leaf area index, low vegetation (`lai_lv`)  | `HTESSSELmon` `lai_lv` | `Lmon` `lai`       |
 | [67.128](https://codes.ecmwf.int/grib/param-db/67) | Leaf area index, high vegetation (`lai_hv`) | `HTESSSELmon` `lai_hv` | `Lmon` `lai`       |

The variables have not been added to the `ACCUMFLD` list in [grib_codes.json](https://github.com/EC-Earth/ece2cmor3/blob/master/ece2cmor3/resources/grib_codes.json), as I thought they shouldn't.

**Comment by treerink** [comment]
The updated `output-control-files` for this request have been commited in the optimesm branch in the ECE repository, see [ECE portal post 1312-111](https://dev.ec-earth.org/issues/1312#note-111) for the comment on the 3 hourly output.

**Comment by treerink** [comment]
However, there is one small issue to solve I realize now: the conflict of the three add scripts:
```
add-htessel-vegetation-variables.sh
add-lpjg-cc-diagnostics.sh
add-variables-for-co2box.sh
```
Currently the latter (the `add-variables-for-co2box.sh`) does not cause a conflict. But the maintenance friendly & robust approach will cause a conflict currently with the modifications applied to  `CMIP6_CV.json` when the two other scripts are run at the same time at cmorisation stage (this did not affect running `genecec`).

**Comment by treerink** [comment]
I post-pone a solution for this for after the back merge of the co2box branch into the master (#792  #785).

**Comment by treerink** [comment]
Last potential issue solved, one could run now:
```
./add-lpjg-cc-diagnostics.sh
 ./add-variables-for-co2box.sh
 ./add-htessel-vegetation-variables.sh no-clean-before
```

**Comment by treerink** [comment]
Added request and comment from the [ECE portal issue 1312-120 post](https://dev.ec-earth.org/issues/1312#note-120):
> The cvl/cvh/lai_lv/lai_hv variables should ideally be saved daily, but monthly is enough for general evaluation.

Addressed in 0082f65.

>> Cmorisation works fine for `lai` and `cv`, but the results for vegetation type (`tv`) are just rubbish. 
>>  (A possible solution could be to replace setgridtype,regular by remapnn for the remapping with cdo)

> actually what is needed is setgridtype,regularnn for the tvl/tvh variables

Addressed in 176a74d.

**Comment by treerink** [comment]
84ea812 fixes a small but essential BUG at cmorisation time.
